### PR TITLE
Fixed styling on users page

### PR DESCRIPTION
**Pull request by kholtzman5610**
- Changed the font-size 1.5rem app/assets/stylesheets/includes/header.scss
- Moved User's script/link into app/views/layouts/application.html.erb

### CODE REVIEW

**Comment by kholtzman5610** [comment]
### **User Login**
SharedScreenshot



### **No User Login**
No-User-Login

**Review by Javier-Jimenez-18 — APPROVED** [review]
User logged in/logged out home page looks good to me. Great job.

Missing http://localhost:3000/users screenshot when logged in as an admin.

**Comment by micronix** [comment]
@kholtzman5610 there are no screenshots of the users page and I also don't see the changes we made to the users page. Does that still need to be commited?

**Comment by kholtzman5610** [comment]
@micronix @Javier-Jimenez-18 screenshot fixed and updated code fixed.

**Review by Javier-Jimenez-18 — APPROVED** [review]
Looks good.  Did we decide to put some more space between the AACT logo and the 2 AACT Users Section in the Users View Page?

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,6 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet"/>
+  <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.8.1/bootstrap-table.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.0/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.8.1/bootstrap-table.min.js"></script>
+
```
Comment (micronix) [inline app/views/layouts/application.html.erb:9]: 1. Remove the old bootstrap version
2. Remove the old jquery version
3. Update the bootstrap-table version from 1.8.1 to the latest one https://cdnjs.com/libraries/bootstrap-table

**Review by micronix — CHANGES_REQUESTED** [review]
Remove/update package versions

**Review by micronix — APPROVED** [review]
Looks good

**Pull request merged**